app/components/elements/AreaSummary/index.jsx

```diff
@@ -10,20 +16,58 @@ class AreaSummary extends React.Component {
     program: PropTypes.string.isRequired,
   }
 
+  constructor(props) {
```

Comment:
You don't need to use a constructor for this

```suggestion
state = {}
```